CHANGELOG.md

```diff
@@ -11,6 +11,13 @@
   Before: `@symfony/ux-dropzone/dropzone`
   After: `symfony--ux-dropzone--dropzone`
 
+* Support for "lazy controllers" was added. By setting the `fetch`
+  in `controllers.json` to `lazy`, your controller will not
+  be downloaded until the controller element first appears on the page.
+
+* The `webpackMode` option in `controllers.json` was deprecated. Use
```

Comment:
We should open a PR on UX and Flex to change this there. Do you have time to do it?

Reply:
I already did! :) https://github.com/symfony/ux/pull/53

Reply:
But flex - I hadn’t thought about that piece - I’ll get that done

Reply:
Flex PR: https://github.com/symfony/flex/pull/735